modules/ROOT/pages/custom-response-policy-example.adoc

```diff
@@ -156,11 +156,12 @@ Now that you have defined the input parameters for your policy, you can start wi
 . Add your dependencies in the `pom.xml` file.
 . Add conditions in the `template.xml` file.
 
+[[add_dependencies_in_pom]]
 === Add Dependencies in POM.XML File
 
-In your `pom.xml`, include all the dependencies that you need to use. In this case, you only need the `http-policy-transform-extension`, which will help you to easily modify the response information.
+To transform your response with the new status code and payload you must create an extension. This helps you to avoid developing the boilerplate code to transform the response message.
```

Comment:
I was thinking that, as we already have this HTTP Policy Transform extension documented, we can include a link to the docs somewhere. The docs are [here](https://docs.mulesoft.com/api-manager/2.x/http-policy-transform). What do you think?

Reply:
Good point! Done!